examples/jsm/renderers/common/extras/PMREMGenerator.js

```diff
@@ -119,21 +119,22 @@ class PMREMGenerator {
 	 * Generates a PMREM from a supplied Scene, which can be faster than using an
 	 * image if networking bandwidth is low. Optional sigma specifies a blur radius
 	 * in radians to be applied to the scene before PMREM generation. Optional near
-	 * and far planes ensure the scene is rendered in its entirety (the cubeCamera
-	 * is placed at the origin).
+	 * and far planes ensure the scene is rendered in its entirety 
+	 * Optional size, the renderTarget default size is 256
+	 * Optional cubeCamera position, the cubeCamera ist default position is the origin
 	 */
-	fromScene( scene, sigma = 0, near = 0.1, far = 100 ) {
+	fromScene( scene, sigma = 0, near = 0.1, far = 100, size = 256, position = new Vector3( 0, 0, 0) ) {
```

Comment:
This looks like a good change to me - I should have added size control to `fromScene` when I made the PMREM size variable in the first place. It can also increase render speed by decreasing cache usage to set the size small when using rougher materials.